### PR TITLE
Automated cco upgrade test case

### PR DESCRIPTION
**Pull request by lwan-wanglin**
Add cloud-credential-operator component upgrade test.

### CODE REVIEW

**Comment by lwan-wanglin** [comment]
@gpei Could you please help to review this one too? thanks.

**Comment by gpei** [comment]
The code logic lgtm.

**Comment by lwan-wanglin** [comment]
@liangxia Please help to review when you have time, thank you in advance!

File: features/upgrade/cloudcredential/upgrade.feature [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Feature: CloudCredentialOperator components upgrade tests
+  @upgrade-prepare
+  @admin
+  # @author lwan@redhat.com
+  # @case_id OCP-34260
```
Comment (liangxia) [inline features/upgrade/cloudcredential/upgrade.feature:5]: case ID should be with upgrade check scenario.

Comment (lwan-wanglin) [reply features/upgrade/cloudcredential/upgrade.feature]: > case ID should be with upgrade check scenario.

Thanks for reviewing, has updated, is this what you mean?

**Pull request merged**